src/views/perms/AssetPermission/AssetPermissionList.vue

```diff
@@ -50,7 +50,7 @@ export default {
         columnsExtra: ['action'],
         columns: [
           'name', 'users_amount', 'user_groups_amount', 'assets_amount', 'nodes_amount',
-          'accounts', 'is_valid', 'is_expired', 'from_ticket', 'is_active', 'actions'
+          'accounts', 'labels', 'is_valid', 'is_expired', 'from_ticket', 'is_active', 'actions'
         ],
         columnsShow: {
           min: ['name', 'actions'],
```

Comment:
To analyze code differences, please share the actual parts of the given code to compare.